database/migrations/sqls/20240803151735-add-closing-date-column-tender-table-up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tender
+ADD COLUMN closing_date DATE DEFAULT CURRENT_DATE;
```

Comment:
Why did you put CURRENT_DATE as the default closing_date?

Reply:
because otherwise the data already entered will have an empty field. But we can get rid of that

Reply:
I see what you are saying. I think leaving CURRENT_DATE is fine.